src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -93,7 +93,12 @@ public void setReplay(Replay replay) {
     timeLabel.setText(timeService.asShortTime(replay.getStartTime()));
     modLabel.setText(replay.getFeaturedMod().getDisplayName());
     playerCountLabel.setText(i18n.number(replay.getTeams().values().stream().mapToInt(List::size).sum()));
-    qualityLabel.setText(i18n.get("percentage", (int) ratingService.calculateQuality(replay) * 100));
+    double gameQuality = ratingService.calculateQuality(replay);
+    if (Double.isNaN(gameQuality)) {
+      qualityLabel.setText(i18n.get("percentage", Math.round(ratingService.calculateQuality(replay) * 100)));
```

Comment:
use gamequality variable

Reply:
(ratingService.calculateQuality(replay) called twice

Reply:
That was the Plan, don't know what happened to that